scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -650,18 +650,22 @@ function main {
               echo
               println DEBUG "# Select wallet to register (only non-registered wallets shown)"
               if [[ ${op_mode} = "online" ]]; then
-                if ! selectWallet "non-reg" "${WALLET_PAY_VK_FILENAME}" "${WALLET_STAKE_VK_FILENAME}"; then
-                  [[ "${dir_name}" != "[Esc] Cancel" ]] && waitForInput; continue
-                fi
+                selectWallet "non-reg" "${WALLET_PAY_VK_FILENAME}" "${WALLET_STAKE_VK_FILENAME}"
+                case $? in
+                  1) waitForInput; continue ;;
+                  2) continue ;;
+                esac
```

Comment:
given the amount of time these 4 lines are repeating, makes sense to convert it to function instead?

Reply:
Can you break out of loop in calling code with continue from within a function call? 🤔

I guess I could try.

Reply:
Though a bit spammy, there is no good way to reduce this code piece as you can't manipulate a loop from a function call. Because of this I will leave it as is for now and merge.